### PR TITLE
Fix to #1271

### LINKED ISSUE #2271: optimization tool: Missing geothermal potential of the scenario. Consider running lake potential script first.
This is an issue posted by @kerembeygo (transferred from #2259)

After running `water-body-potential` tool, the `optimization` tool raises this error. (NOTE: the error message needs to be updated to reflect the name change in the tool. Also, the input files system should have caught this first!)

### PR DESCRIPTION
**Pull request by JIMENOFONSECA**
@daren-thomas this should fix #2271 let me know if this works

### CODE REVIEW

File: cea/optimization/optimization_main.py [hunk 1]
```diff
@@ -147,10 +150,10 @@ def check_input_files(config, locator):
         raise ValueError(
             "Missing sewage potential of the scenario. Consider running sewage heat exchanger script first.")
     if not os.path.exists(locator.get_lake_potential()):
-        raise ValueError("Missing lake potential of the scenario. Consider running lake potential script first.")
+        raise ValueError("Missing water-body potential of the scenario. Consider running lake potential script first.")
```
Comment (daren-thomas) [inline cea/optimization/optimization_main.py]: "Consider running _lake potential_ script first"... I think only half of that error message was updated. Additionally, I think the locator method (`get_lake_potential`) could also be updated to `get_water_body_potential`.

File: cea/scripts.yml [hunk 2]
```diff
@@ -248,8 +248,15 @@ Optimization:
     description: Run optimization for the given scenario
     interfaces: [cli, arcgis, dashboard]
     module: cea.optimization.optimization_main
-    parameters: ['general:scenario', 'thermal-network:network-type', 'thermal-network:network-names',
-                 'optimization', 'decentralized', 'general:multiprocessing']
+    parameters: ['general:scenario', 'general:multiprocessing', optimization]
+    input-files:
+      - [get_street_network]
+      - [get_total_demand]
+      - [get_lake_potential]
```
Comment (daren-thomas) [inline cea/scripts.yml]: note, when updating the InputLocator method this would need to change too...

**Comment by daren-thomas** [comment]
@JIMENOFONSECA there are still some small changes here for you to make before we can merge.

File: cea/optimization/optimization_main.py (hunk 1 above)
Comment (JIMENOFONSECA) [reply cea/optimization/optimization_main.py]: done!

**Comment by JIMENOFONSECA** [comment]
@daren-thomas please merge if the PR is ok now, thanks

**Pull request merged**